src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheck.java

```diff
@@ -496,6 +496,51 @@
  * public class SomeClass { }
  * </pre>
  * <p>
+ * To configure the Check with groups of static imports when staticGroups=&quot;&quot; represents
+ * only one group. In below example the static imports {@code com...} should be before
+ * {@code java...}:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;ImportOrder&quot;&gt;
+ *   &lt;property name=&quot;staticGroups&quot; value=&quot;&quot;/&gt;
+ *   &lt;property name=&quot;sortStaticImportsAlphabetically&quot; value=&quot;true&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <pre>
+ * import static java.io.File.listRoots;
+ * import static javax.swing.WindowConstants.*; // OK
+ * import static java.io.File.createTempFile; // violation should be before javax
+ * import static com.puppycrawl.tools.checkstyle.checks.imports.avoidstaticimport
+ * .InputAvoidStaticImportNestedClass.InnerClass;  // violation should be before java and javax
```

Comment:
all long lines should be updated to smth like:
`import static com.mycompany.MyClass`

in our Inputs we use long names only because we demand code to be compilable, in users doc we can use any imports.

Reply:
@romani Look into this u will get what I am trying to say. https://github.com/checkstyle/checkstyle/issues/5982#issuecomment-813654853

Reply:
Replied, you can fix it in separate PR